README.md

```diff
@@ -77,10 +77,12 @@ $ aegir-test browser
 $ gulp test:browser
 ```
 
-If you want to run tests in a webworker you can also pass `--webworker` as a flag to enable that.
+Browser tests by default run inside the main thread and a webworker thread. If you want to disable or only run your tests in a webworker, you can do it by passing the flag:
 
-If the needed environment variables are set, tests are also run on [Sauce Labs].
-You will need
+- `--webworker=false`
+- `--webworker=only`
```

Comment:
@diasdavid 

>I'm also not convinced that we should run in webworkers by default

The reason I went with this is because a) most of our code *needs* to run in WW either way[1] and b) it was a lot less work to enable them by default rather than selectively enable/disable them. 

Any specific reason as to why this is not the right path?

1 - We have a lot less code that would not run in a WW than otherwise, the ones that come to mind that should not be enabled in WW/Browsers are the transports and storage engines that are not natively supported by the browser, the rest should work across the board, and browser tests are already disabled for those either way, so the only one left that should not be enabled in WW is WebRTC.